src/components/dropdown-select.js

```diff
@@ -214,7 +214,7 @@ const DropdownSelect = ({
       }
     }
     setSelectedOptions(newSelectedOptions)
-    onSelect(newSelectedOptions)
+    onSelect({ value: newSelectedOptions, i })
```

Comment:
lets just do ...newSelectedOptions so we don't have to access them by value.title / value.description

Reply:
let me know what you think: https://github.com/EQWorks/lumen-labs/blob/64ecf96522719bdc695f6b9d244ac1a617d637bb/src/components/dropdown-select.js#L217